README.md

```diff
@@ -16,6 +16,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Code Tuts+](https://code.tutsplus.com/articles/call-for-authors-write-for-tuts--cms-22034) - $100 (Quick tip) $250 (Tutorial)
   > Technical focused articles. Pick from a list of possible articles.
 
+- [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - unknown
+  > Technical focused articles. specific on Flutter.
```

Comment:
Hey, thanks for adding this. Grammatical error here, should be "Specific". Otherwise, happy to merge after you make that change.